composer.lock

```diff
@@ -1517,5 +1517,5 @@
     "prefer-lowest": false,
     "platform": [],
     "platform-dev": [],
-    "plugin-api-version": "2.1.0"
+    "plugin-api-version": "2.0.0"
```

Comment:
Is this change intentional? There are no changes to `composer.json` in this PR and this is an auto-generated file.

Reply:
@adekbadek Nope. It might be result of how the requirement is written and running a `composer update`. If you prefer I can checkout the lock file, but I don't know what this library even does so I didn't intentionally update it. Lmk.

Reply:
Let's skip it for this PR.